src/components/navLink.js

```diff
@@ -0,0 +1,15 @@
+import { Route, Routes } from 'react-router-dom';
+import Missions from './missions';
+import MyProfile from './myProfile';
+import Rockets from './rockets';
+
+const NavLink = () => (
+  <main className="app-container">
+    <Routes>
+      <Route path="/rockets" element={<Rockets />} />
```

Comment:
Here can you add empty route action (/) above (/rockets) link
(<Route path="/" element={<Rockets />} /> ) and  below (/myprofile) add not found route action ('*) like <Route path="*" element={<NotFound />} />